src/views/Gamefi/Gamefi.tsx

```diff
@@ -1,43 +1,284 @@
+        const apr = isActive ? getFarmApr(new BigNumber(farm.poolWeight), cakePrice, totalLiquidity) : 0
+
+        return { ...farm, apr, liquidity: totalLiquidity }
+      })
+
+      if (query) {
+        const lowercaseQuery = latinise(query.toLowerCase())
+        farmsToDisplayWithAPR = farmsToDisplayWithAPR.filter((farm: FarmWithStakedValue) => {
+          return (
+            latinise(farm.lpSymbol.toLowerCase()).includes(lowercaseQuery) ||
+            latinise(farm.quoteToken.symbol.toLowerCase()).includes(lowercaseQuery)
+          )
+        })
+      }
+      return farmsToDisplayWithAPR
+    },
+    [cakePrice, query, isActive],
+  )
+
+  const pools = useMemo(() => {
+    const cakePool = poolsWithoutAutoVault.map((pool) => pool.sousId === 0)
+    const cakeAutoVault = { ...cakePool, isAutoVault: true }
+
+    return [...poolsWithoutAutoVault]
+  }, [poolsWithoutAutoVault])
+
+  const [finishedPools, openPools] = useMemo(() => partition(pools, (pool) => pool.isFinished), [pools])
+  const stakedOnlyFinishedPools = useMemo(
+    () =>
+      finishedPools.filter((pool) => {
+        if (pool.isAutoVault) {
+          return accountHasVaultShares
+        }
+        return pool.userData && new BigNumber(pool.userData.stakedBalance).isGreaterThan(0)
+      }),
+    [finishedPools, accountHasVaultShares],
+  )
+  const stakedOnlyOpenPools = useMemo(
+    () =>
+      openPools.filter((pool) => {
+        if (pool.isAutoVault) {
+          return accountHasVaultShares
+        }
+        return pool.userData && new BigNumber(pool.userData.stakedBalance).isGreaterThan(0)
+      }),
+    [openPools, accountHasVaultShares],
+  )
+
+  const stakedMemoized = useMemo(() => {
+    const stakingList = { activeFarms: [], inactiveFarms: [], activePools: [], inactivePools: [] }
+
+    if (stakedOnly) {
+      stakingList.activeFarms = farmsList(stakedOnlyFarms)
+      stakingList.inactiveFarms = farmsList(stakedInactiveFarms)
+      stakingList.activePools = stakedOnlyOpenPools
+      stakingList.inactivePools = stakedOnlyFinishedPools
+    } else {
+      stakingList.activeFarms = farmsList(activeFarms)
+      stakingList.inactiveFarms = farmsList(inactiveFarms)
+      stakingList.activePools = openPools
+      stakingList.inactivePools = finishedPools
+    }
+    return stakingList
+  }, [
+    farmsList,
+    activeFarms,
+    inactiveFarms,
+    stakedInactiveFarms,
+    stakedOnly,
+    stakedOnlyFarms,
+    finishedPools,
+    openPools,
+    stakedOnlyOpenPools,
+    stakedOnlyFinishedPools,
+  ])
+
+  const renderContent = ({ RENDER_TYPE }: { RENDER_TYPE?: string }): JSX.Element => {
+    const render = (type) => {
+      switch (type) {
+        case 'RENDER_ENDED':
+          return stakedMemoized.inactiveFarms.length !== 0 && stakedMemoized.inactivePools.length !== 0 ? (
+            <>
+              {stakedMemoized.inactiveFarms.length !== 0 ? (
+                stakedMemoized.inactiveFarms.map((farm) => (
```

Comment:
halaka @kojiadrianojr sabi ko baga iwasan ang nested ternary operation

Reply:
Kuys si dati yan na ternary and I believe na discuss na natin to before if para saan siya. 
Di man siya ternary na bawal since if ever, ma throw yan ki lint error. 
Duwa kaya irerender kayan,
1. check kang state if empty = to display isa lang na error message
2. Check kung inactive or active yung wala para if empty either, malabas si no launches